assets/js/18f-guide.js

```diff
@@ -0,0 +1,11 @@
+$(function() {
+  $('.sidebar-nav').each(function() {
+    accordion($(this));
+  });
+});
```

Comment:
Ideally (in my opinion) the off-the-shelf Web Design Standards could be used, so was trying to move the application-specific code out of `accordion.js`. That file still has your tweak to the `Accordion` constructor, though.